src/client/home/actions/index.ts

```diff
@@ -41,9 +41,9 @@ const loadStats = () => (
     home: { statistics },
   } = getState()
   if (
-    statistics.userCount === null ||
-    statistics.linkCount === null ||
-    statistics.clickCount === null
+    statistics.userCount === 0 ||
+    statistics.linkCount === 0 ||
+    statistics.clickCount === 0
```

Comment:
just to check: back then when this check was for null, its purpose was to see if this `loadStats`  function ran before right? if that's the case, this time you have refactored the `useEffect` to only run once so you wont ever need any check here at all

Reply:
make sense, removed the checks!